docs/tokens/token_list.md

```diff
@@ -24,43 +24,28 @@ In addition to NEON and wNEON tokens, below are some common tokens that are depl
 
 ---
 
-|Token|Mainnet Address                           |
+|Token|Mainnet ERC20 Wrapper Address                        |
 |-----|:-----------------------------------------|
-|USDC |Comming soon...                           |
-|USDT |Comming soon...                           |
-|wSOL |Comming soon...                           |
+|NEON| 0x34260f162a1CcB3ad6c2D974d8e330f54b2c5345|
+|USDC |0x512E48836Cd42F3eB6f50CEd9ffD81E0a7F15103                          |
+|USDT |0x6eEf939FC6e2B3F440dCbB72Ea81Cd63B5a519A5                           |
+|wSOL |0xc7Fc9b46e479c5Cb42f6C458D1881e55E6B7986c                           |
   </TabItem>
   <TabItem value="devnet" label="Devnet">
 
-|Token|Devnet Address                            |
+|Token|Devnet ERC20 Wrapper Address                           |
 |-----|:-----------------------------------------|
-|WNEON|0x11adC2d986E334137b9ad0a0F290771F31e9517F|
 |USDT |0x6eEf939FC6e2B3F440dCbB72Ea81Cd63B5a519A5|
-|wDAI |0xC989FfF9d02C459117A2EaBb8dA3f03DFd9045DE|
 |USDC |0x512E48836Cd42F3eB6f50CEd9ffD81E0a7F15103|
+|NEON|0x11adC2d986E334137b9ad0a0F290771F31e9517F|
```

Comment:
The address in the [source of truth](https://www.notion.so/neonlabs/8b7acf3edab94551a2630db4665adbe4?v=cf71adde2ee742fb898daf6235a7f5d5) has a very different format {89dre8rZjLNft7HoupGiyxu3MNftR577ZYu8bHe2kK7g} -- I am not comfortable updating these to that -- please can you confirm @anton-lisanin ?

Reply:
This is not correct address

@YYYurchenko fix [source of truth](https://www.notion.so/neonlabs/8b7acf3edab94551a2630db4665adbe4?v=cf71adde2ee742fb898daf6235a7f5d5)

Reply:
The final source of truth is here https://github.com/neonlabsorg/token-list/blob/main/tokenlist.json

Reply:
done